templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -177,14 +177,14 @@ objects:
         level: debug
         color: true
       openshift:
-        host: ${CLUSTER_AUTH_HOST}
-        ca_file: ${CA_FILE}
-        bearer_token_file: ${BEARER_TOKEN_FILE}
-        image_pull_policy: ${IMAGE_PULL_POLICY}
+        host: "${CLUSTER_AUTH_HOST}"
+        ca_file: "{CA_FILE}"
```

Comment:
You removed the $ from {CA_FILE} this causes the ca_file setting to be "{CA_FILE}" which is not correct.